File: sdkwork-ai-sdk-java/src/main/java/com/sdkwork/ai/model/EmbeddingRequest.java
package com.sdkwork.ai.model;

public class EmbeddingRequest {
    private String model;
    private Object input;
    private Integer dimensions;
    private String user;
    private String encodingFormat;

    public String getModel() {
        return this.model;
    }
    
    public void setModel(String model) {
        this.model = model;
    }

    public Object getInput() {
        return this.input;
    }
    
    public void setInput(Object input) {
        this.input = input;
    }

    public Integer getDimensions() {
        return this.dimensions;
    }
    
    public void setDimensions(Integer dimensions) {
        this.dimensions = dimensions;
    }

    public String getUser() {
        return this.user;
    }
    
    public void setUser(String user) {
        this.user = user;
    }

    public String getEncodingFormat() {
        return this.encodingFormat;
    }
    
    public void setEncodingFormat(String encodingFormat) {
        this.encodingFormat = encodingFormat;
    }
}

File: sdkwork-ai-sdk-csharp/Models/EmbeddingRequest.cs
using System;
using System.Collections.Generic;
using System.Text.Json.Serialization;

namespace Ai.Models
{
    public class EmbeddingRequest
    {
        public string? Model { get; set; }
        public object? Input { get; set; }
        public int? Dimensions { get; set; }
        public string? User { get; set; }
        public string? EncodingFormat { get; set; }
    }
}
